site/content/docs/getting-further/is-my-app-testable.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Is My App Security Testable?"
+type: page
+tags: 
+- advanced
+EditableContent: true
+addBreadcrumbs: true
+---
+Web app security is hard, and there are many, many aspects to it (see [OWASP](https://www.owasp.org) for more details).
+
+When it comes to security testing the best way to test the security of a web app is via a pentest conducted by experts.
+However such pentests are expensive, and I have not heard of any company that can afford to carry out
+regular pentests on all of their web apps.
+
+This is where ZAP can shine. ZAP is a cheap and effective way to test all of your web apps, whenever you need them tested.
+
+ZAP can help you secure your web apps, but you can make life much more difficult for ZAP, and that in turn makes life more difficult for you.
+
+Making it harder for ZAP to test your web apps will not make them more secure, it can in fact make them less secure.
+If you make automated testing too hard then the race to find vulnerabilities in your apps will be between
+the pentesters you hire and the malicious attackers who are out to compromise you.
```

Comment:
It also adds complexity and functionality that may also present attack surface or maintenance burden.

Reply:
I dont think we should get into that argument tbh - if people choose to add these controls to prod then thats their choice, we just need to try to encourage them to remove them from their testing env.